services/src/main/java/org/exoplatform/addons/jwt/JwtFilter.java

```diff
@@ -59,9 +63,17 @@ public JwtFilter() {
     if (exclusions != null) {
       exclusionsUrl.addAll(Arrays.asList(exclusions.split(",")));
     } else {
-      exclusionsUrl.add("/portal/rest/onlyoffice/editor/status/");
-      exclusionsUrl.add("/portal/rest/onlyoffice/editor/content/");
+      exclusionsUrl.add("/portal/rest/onlyoffice/editor/status/.*");
+      exclusionsUrl.add("/portal/rest/onlyoffice/editor/content/.*");
+      exclusionsUrl.add("/portal/rest/v1/social/users/[0-9]*/avatar");
```

Comment:
```suggestion
      exclusionsUrl.add("/portal/rest/v1/social/users/[0-9]*/avatar.*");
      exclusionsUrl.add("/portal/rest/v1/social/spaces/[0-9]*/avatar.*");
```
Do we need to consider lastModified get param and add spaces avatars WDYT?

Reply:
lastModified are not part of httpRequest.getUri. 

space avatar are not displayed to anonymous user, specially in mail notifications, so no need to add this url

Reply:
Thanks for the clarification